gateway/src/apicast/policy_loader.lua

```diff
@@ -211,16 +214,24 @@ local mt = {
   __call = function(loader, ...) return loader.env.require(...) end
 }
 
-function _M.new(name, version, dir)
-  local apicast_dir = dir or getenv('APICAST_DIR') or '.'
+do
```

Comment:
Why is this in a `do` block?

Reply:
It makes a variable that is not really needed to be exposed for the rest of the module. 

I've seen this in resty-core and other resty libraries. I think it is quite nice pattern to create temporary variables.

Reply:
I see :+1: